### PR TITLE
Documentation + Readability Changes 

### PR DESCRIPTION
**Pull request by ArtSabintsev**
- Cleaned up the documentation
- Only other documentation is around having video example tutorials in as many places as possible.

### CODE REVIEW

File: doc/guides/quickstart.md [hunk 1]
```diff
@@ -27,34 +25,35 @@ const rpcProvider = new HttpRpcProvider(dispatchers[0]);
...
 transactionSender = transactionSender as TransactionSender
 ```
 
-## How to send balance using Pocket
+### **Performing Transactions**
+
+Afterward importing the sender's account and creating a transaction sender object, any of the transactions below can be performed.
 
-To send a balance we add 1 step to the whole process and that's it.
+### **Send a balance**
```
Comment (Evalir) [inline doc/guides/quickstart.md:54]: I think this title should be "Sending POKT". What do you think @pabelnl

**Review by Evalir — DISMISSED** [review]
Looks good to me in general! @pabelnl should review further but just gave it a quick read.

File: doc/guides/quickstart.md (hunk 1 above)
Comment (pabelnl) [reply doc/guides/quickstart.md:54]: "Send a transaction" seems better

Comment (pabelnl) [inline doc/guides/quickstart.md:54]: Change` ### **Send a balance**` to `### **Send a balance transaction**`

Comment (ArtSabintsev) [reply doc/guides/quickstart.md:54]: Done. Just pushed it up.

**Comment by Evalir** [comment]
@pabelnl wonder if this is good to merge?

**Review by pabelnl — APPROVED** [review]

**Pull request merged**